core/ui/src/views/NodeFirewall.vue

```diff
@@ -202,18 +202,17 @@
                       </cv-data-table-cell>
                       <cv-data-table-cell>
                         <span v-if="row.addresses.length <= 3">
-                          <span v-for="(addr, idx) in row.addresses" :key="idx">
-                            {{ addr
-                            }}<br v-if="idx < row.addresses.length - 1" />
-                          </span>
+                          <div v-for="(addr, idx) in row.addresses" :key="idx">
+                            {{ addr }}
+                          </div>
                         </span>
                         <span v-else>
-                          <span
+                          <div
                             v-for="(addr, idx) in row.addresses.slice(0, 3)"
                             :key="idx"
                           >
                             {{ addr }}<br />
```

Comment:
`<div>` elements go to a new line by default

```suggestion
                            {{ addr }}
```